git_services/tests/conftest.py

```diff
@@ -36,18 +37,28 @@ def _commit_everything():
 
 @pytest.fixture
 def make_branch():
-    def _make_branch(git_cli, name):
+    def _make_branch(git_cli: GitCLI, name: str):
         git_cli.git_checkout(f"-b {name}")
 
     return _make_branch
 
 
 @pytest.fixture
-def init_git_repo(git_cli, create_file, commit_everything):
-    def _init_git_repo():
+def init_git_repo(git_cli: GitCLI, create_file, commit_everything):
+    def _init_git_repo(init_renku=False):
         git_cli.git_init()
         git_cli.git_config("user.name 'Test User'")
         git_cli.git_config("user.email 'test.user@renku.ch'")
+        if init_renku:
+            check_call(
+                args=[
+                    "renku",
+                    "init",
+                    "--template-id=minimal",
+                    ".",
+                ],
+                cwd=git_cli.repo_directory,
+            )
```

Comment:
```suggestion
            check_call(
                args=[
                    "renku",
                    "init",
                    "--template-id=minimal",
                    git_cli.repo_directory,
                ],
            )
```
since path is an argument to init already.

Reply:
Done.